docs/create-pyenv.md

```diff
@@ -215,20 +227,42 @@ virtualenv.
       runtime_library_dirs = /usr/local/opt/openblas/lib
       ```
 
+   To link numpy to Accelerate, create a `~/.numpy-site.cfg` file like
+   this (or go without the file since Accelerate is now the default),
+   and remember to run `pip install <<packages>> --no-binary numpy` in the
+   pip-install steps below.
+
+      ```
+      [accelerate]
+      libraries = Accelerate, vecLib
+      ```
+
+   In late 2023, numpy binary wheels are expected to use Accelerate on macOS 1.14+.
+
+   **Recommendation:** On macOS 13.3+, use
+   `pip install numpy==1.24.3 --no-binary numpy`
+   to link numpy to the macOS Accelerate library. Otherwise use
+   `pip install numpy==1.24.3`
+   to use numpy's built-in copy of OpenBLAS.
```

Comment:
I'm a little confused on the wording in this section (starting with line 242) vs line 192 above - are we recommending Mac users to use Accelerate? Or still use OpenBLAS for now?

Reply:
Good catch! I'll remove this until the Numpy team makes it work properly on Accelerate.